### PR TITLE
Upgraded neo to 1.10.0

### PR DESCRIPTION
**Pull request by RephaelCongmon**
*Description of changes:*

Modified patch to work with 1.10.0 recipe.
Upgraded Neo to 1.10.0

### CODE REVIEW

File: recipes-devtools/amazon-sagemaker-neo/neo-ai-dlr_1.7.0.bb [hunk 1]
```diff
@@ -74,10 +79,10 @@ do_install() {
     distutils3_do_install
 
     # setup.py install some libs under datadir, but we don't need them, so remove.
-    rm ${D}${datadir}/dlr/*.so
+   # rm ${D}${datadir}/dlr/*.so
```
Comment (rpcme) [inline recipes-devtools/amazon-sagemaker-neo/neo-ai-dlr_1.7.0.bb:82]: Please remove this line if it's unnecessary

**Review by rpcme — COMMENTED** [review]
Hi, there are a couple housekeeping items commented inline. Usually we will want to remove commented code (as opposed to comments themselves) to reduce uncertainty

File: recipes-devtools/amazon-sagemaker-neo/files/0002-CMakeLists_remove_test_file_downloads.patch [hunk 2]
```diff
@@ -156,3 +213,6 @@ index c7993b9..87fd6a1 100644
  endif()
  # Group sources
  #auto_source_group("${SOURCES}")
```
[the remark this answers is not in the document]
Comment (nateglims) [reply recipes-devtools/amazon-sagemaker-neo/files/0002-CMakeLists_remove_test_file_downloads.patch]: This is a patch though

**Comment by nateglims** [comment]
Build is failing because the patch does not apply now:


```
ERROR: neo-ai-dlr-1.10.0-r0 do_patch: Applying patch '0002-CMakeLists_remove_test_file_downloads.patch' on target directory '/tmp/build/tmp/work/core2-64-poky-linux/neo-ai-dlr/1.10.0-r0/git'
--
Patch 0002-CMakeLists_remove_test_file_downloads.patch does not apply (enforce with -f)
```

File: recipes-devtools/amazon-sagemaker-neo/files/0002-CMakeLists_remove_test_file_downloads.patch (hunk 2 above)
[the remark this answers is not in the document]
Comment (nateglims) [reply recipes-devtools/amazon-sagemaker-neo/files/0002-CMakeLists_remove_test_file_downloads.patch]: Removing the line from the diff doesn't remove the line in patching. It just messes up the diff algorithm.

[the remark this answers is not in the document]
Comment (nateglims) [reply recipes-devtools/amazon-sagemaker-neo/files/0002-CMakeLists_remove_test_file_downloads.patch]: Removing the line in patching has no impact to the build, but sends the signal that we thought it was important. The diff should do the minimal thing required.

**Review by nateglims — APPROVED** [review]

**Pull request merged**